### PR TITLE
Fix naming of a few @Place files.

### PR DESCRIPTION
**Pull request by hotelvictorcharlie**
A few `@Place` files are incorrectly named, and one probably shouldn't be there at all.

### CODE REVIEW

File: EverDrive Pack SMDBs/MiSTer.txt [hunk 1]
```diff
@@ -1757,7 +1757,6 @@ d3f8eb881e2560cdce00738c716b9ad679128cb827493c267594f26c1c0ac744	MiSTer/Games/Ja
 ae81e351200fabd19fb96831237514259f27bc38e692888567dca7e17dd9a901	MiSTer/Games/Jupiter/Jupiter Ace - Games - [ACE] (TOSEC-v2014-08-11_CM).zip	99cb6820c89ef08953ed75457c2e9c4967b8fcdd	e80d7f7c8f999e95c8e927557d25f513	d64428a9
 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/Lynx/@Place Lynx Gamepack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
 c26a36c1990bcf841155e5a6fea4d2ee1a4d53b3cc772e70f257a962ad43b383	MiSTer/Games/Lynx/boot.rom	e4ed47fae31693e016b081c6bda48da5b70d7ccb	fcd403db69f54290b51035d82f835e7b	0d973c9d
-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/MSX/@Place MSX Add-on Pack Here.txt	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
```
Comment (hotelvictorcharlie) [inline EverDrive Pack SMDBs/MiSTer.txt:1760]: The rest of the add-on packs don't have `@Place` files, so I think we can remove this. In any case, it should not end in `.txt`.

File: EverDrive Pack SMDBs/MiSTer.txt [hunk 2]
```diff
@@ -2110,7 +2109,7 @@ d10a59d957b10b6a6f1be908a34fa14515fb967972b82ef3005b91e65aa3545c	MiSTer/Games/QL
 2336aaed0742fb86723b3a45a6a9ba5b227115a7627ed3c219fb64f29bad1366	MiSTer/Games/QL/boot.rom	113d506c413ebe89397fff9d6edfd6ea5969c5b2	9a68b8775c669e941b6547482f5d472e	c6018df3
 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/SMS/Game Gear/@Place EverDrive GG Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/SMS/Master System/@Place Master EverDrive Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/SNES/Place SNES Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
+e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/SNES/@Place SNES Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
```
Comment (hotelvictorcharlie) [inline EverDrive Pack SMDBs/MiSTer.txt:2112]: Was missing the `@`.

File: EverDrive Pack SMDBs/MiSTer.txt [hunk 3]
```diff
@@ -2136,7 +2135,7 @@ ddafc8395ebb374bf3999c408b24c32669eb9ae3183bb24da5fd5fc42a4f6da2	MiSTer/Games/VI
 bf96398a7b0c3c63f09510baaa55900cd0eb8a7520aeeb32131d03c19747d614	MiSTer/Games/VIC20/VIC20 Games [D64] (TOSEC.2016.11.11).zip	a42b45c6d996fa6f262aa3eb8e6b976e6b70a731	b14c293bdf6f9a6d129ad6eca9d81ebc	d678da4e
 d989d941e2d5b4e77a223eaf4fc56b19cd3023dad4aa3ad906ca2b34555eab75	MiSTer/Games/VIC20/VIC20 Games [TAP] (TOSEC.2016.11.11).zip	c25346562a4d0c05a15d15dad00baf787d25712e	a98fe111cd74c908ce781ab0341c53db	facfd257
 537274de07390d3f5f1cd9c84010aada77bc37f3eed105ae912c7cedbd3fdc95	MiSTer/Games/Vector06/Vector06.zip	805da8706c21327fbab2380b949490d6e8fb9696	cb4aa0590755599086fc854e00456297	36d7d54c
-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/Vectrex/Place Vectrex Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
+e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855	MiSTer/Games/Vectrex/@Place Vectrex Pack Here	da39a3ee5e6b4b0d3255bfef95601890afd80709	d41d8cd98f00b204e9800998ecf8427e	00000000
```
Comment (hotelvictorcharlie) [inline EverDrive Pack SMDBs/MiSTer.txt:2138]: Was missing the `@`.

File: EverDrive Pack SMDBs/MiSTer.txt (hunk 1 above)
Comment (hotelvictorcharlie) [inline EverDrive Pack SMDBs/MiSTer.txt:1760]: The rest of the add-on packs don't have `@Place` files, so I think we can remove this. It should also not end in `.txt`.

**Comment by tjanas** [comment]
@hotelvictorcharlie If we're going to remove the MSX placeholder (which is the only file here in the Games/MSX directory), then why not remove the sole placeholder for Games/SNES and others like it too?

**Comment by tjanas** [comment]
Having all the add-ons live inside the MiSTer pack makes it much more unwieldy to manage, especially since each may have a different source for validation and datting purposes. Perhaps not a bad idea to add more placeholders with the correct directory names expected by the MiSTer cores.

**Comment by hotelvictorcharlie** [comment]
Some of the add-on packs only have a very small number (1-5) of files in them ([ao486](https://github.com/frederic-mahe/Hardware-Target-Game-Database/blob/4389d6272786145bd6eb5cf30a5849ea1086b496/EverDrive%20Pack%20SMDBs/MiSTer%20ao486%20Add-On.txt), [TSConf](https://github.com/frederic-mahe/Hardware-Target-Game-Database/blob/4389d6272786145bd6eb5cf30a5849ea1086b496/EverDrive%20Pack%20SMDBs/MiSTer%20TSConf%20Add-On.txt), [MSX](https://github.com/frederic-mahe/Hardware-Target-Game-Database/blob/4389d6272786145bd6eb5cf30a5849ea1086b496/EverDrive%20Pack%20SMDBs/MiSTer%20MSX%20Add-On.txt)) so I think we could at least put those ones into the main MiSTer pack without it getting unwieldy, right?

As for the others, what if we convert them into standalone packs that can be referenced via placeholders from any pack? This could prove useful if we want to create a pack for e.g. the Analogue Pocket (which, as the owner of a Pocket, I think we definitely should 😄 ). Also it would be a good opportunity to close #717 and #718.

**Comment by hotelvictorcharlie** [comment]
Plus it would solve #697 as well.

**Comment by frederic-mahe** [comment]
> Having all the add-ons live inside the MiSTer pack makes it much more unwieldy to manage, especially since each may have a different source for validation and dating purposes. Perhaps not a bad idea to add more placeholders with the correct directory names expected by the MiSTer cores.

I agree. We need more placeholders, for example for the Intellivision, or for the Atari 2600 (although there seems to be a set of 'MiSTer Fixed' ROMs for the Atari 2600. I don't know if this is still relevant).

**Comment by tjanas** [comment]
The "MiSTer Fixed" entries should be reviewed. Most likely they match no-intro now.

**Comment by tjanas** [comment]
Fixed via https://github.com/frederic-mahe/Hardware-Target-Game-Database/pull/971

**Pull request closed without merging**